src/layout/Grid.js

```diff
@@ -87,7 +87,9 @@
         return this;
     };
 
-    Grid.prototype.getCell = function (row, col) {
+    Grid.prototype.getCell = function (_row, _col) {
+        var row = _row * this.cellDensity();
+        var col = _col * this.cellDensity();
```

Comment:
@GordonSmith Actually this does break the ability to drop a widget ontop of another widget to swap their locations in designMode.

When the cellDensity is changed it is automatically applied to each cell's gridRow, gridCol, gridRowSpan and gridColSpan. Lines 91 and 92 shouldn't be necessary.